public_html/wp-content/themes/pattern-directory/css/components/_category-menu.scss

```
@@ -0,0 +1,89 @@
+$grid-unit: 1.5rem;
+$font-weight: 500;
+$font-size: 14px;
```

Also, I've been trying to keep everything in `rem`s for better accessibility - so this should be:

```suggestion
$font-size: 0.875rem;
```

---

Yep, i'll update the px references.